src/BaseCharts.js

```diff
@@ -88,18 +137,78 @@ export function generateChart(chartId, chartType) {
   }
 }
 
-export const Bar = generateChart('bar-chart', 'bar')
+export const Bar = generateChart(
```

Comment:
Need to mark those as pure

https://github.com/reactchartjs/react-chartjs-2/blob/master/src/typedCharts.tsx#L33

Reply:
marked